server/chat-commands/core.ts

```diff
@@ -225,47 +229,42 @@ export const commands: Chat.ChatCommands = {
 
 		return target;
 	},
-	mehelp: [`/me [action] - Adds the given [action] into chat, attributed to the user.`],
+	mehelp: [
+ `/me [action] - Adds the user and attributes the optional [action] to them.`,
```

Comment:
What does "adds the user" mean?

Reply:
The goal was to describe that the action is optional, and what happens when no action is input, while minimally changing from the original wording. 

If it's confusing, something else is probably better.

I'll put something better if it comes to me. If not, I'm down for suggestions.

Reply:
I think the original text is fine as is, but it's missing a clarification on what "adding an action" means. That clarification is the whole roleplaying stuff.